frontend-public/webpack.parts.js

```diff
@@ -1,11 +1,11 @@
 const webpack = require("webpack");
 const autoprefixer = require("autoprefixer");
-const cssnano = require("cssnano")({ zindex: false });
+const cssnano = require("cssnano");
 
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
-const UglifyWebpackPlugin = require("uglifyjs-webpack-plugin");
-const CleanWebpackPlugin = require("clean-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
```

Comment:
Switched from Uglify to Terser plugin. Mainly based on recommendations. Terser has support for ES6 as well, which is where I was running into webpack pipeline errors during the upgrade.

Reply:
Curious if u checked out the parallel plugin on it
https://www.npmjs.com/package/terser-webpack-plugin#parallel

Reply:
Not yet, I'll take a look.

Reply:
Belay that, the options for Terser were made to be compatible from Uglify; I had copied over the parallel option into the configuration during first migration.